app/assets/javascripts/sequence_show.js

```diff
@@ -1,7 +1,29 @@
-import {addCopy, logToGoogle, triggerDownloadModal} from "./utils.js";
+import {addCopy, logToGoogle, triggerDownloadModal, stringTitleize, numberToPercent} from "./utils.js";
 import {showInfoModal} from "./modal.js";
-
```

Comment:
@bmesuere , All pannels are now panels

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13911) by @beardhatcode on Fri Mar 02 2018 at 09:49._